### PR TITLE
and pattern examples & links to R22 notes

### PR DESCRIPTION
**Pull request by jeffvg**
Fixes # .

#### Changes proposed in this Pull Request:
- update with design pattern example
- 
- 

#### To Test:
-

### CODE REVIEW

File: src/docs/release-notes/R22/R22.mdx [hunk 1]
```diff
@@ -33,7 +37,7 @@ The Power Xpert Blue "PX Blue" design system rebranded to Brightlayer User Inter
 -   @brightlayer-ui/icons v1.8.1 ([Changelog](https://github.com/brightlayer-ui/icons/blob/master/CHANGELOG.md))
 -   @brightlayer-ui/icons-svg v1.8.1 ([Changelog](https://github.com/brightlayer-ui/icons/blob/master/CHANGELOG.md))
 -   @brightlayer-ui/icons-mui v2.8.1 ([Changelog](https://github.com/brightlayer-ui/icons/blob/master/mui/CHANGELOG.md))
--   @brightlayer-ui/react-native-vector-icons v2.8.1 ([Changelog](https://github.com/brightlayer-ui/icons/releases))
+-   @brightlayer-ui/react-native-vector-icons v1.4.1 ([Changelog](https://github.com/brightlayer-ui/icons/blob/dev/rn-vector/CHANGELOG.md))
```
Comment (huayunh) [inline src/docs/release-notes/R22/R22.mdx:40]: should be pointing to changelog in master branch

**Comment by huayunh** [comment]
Would you mind editing `package.json`'s `version` field to make it say 3.0.0 in this PR too? Sorry I missed that earlier.

File: src/docs/release-notes/R22/R22.mdx [hunk 2]
```diff
@@ -15,8 +15,12 @@ The Power Xpert Blue "PX Blue" design system rebranded to Brightlayer User Inter
 ### Documentation Site
 
 -   Added [migration instructions](https://brightlayer-ui.github.io/migration) for package migration
--   Added new design patterns for Angular and React
--   Updated design patterns to be responsive
+
+### Design Pattern Examples
```
Comment (huayunh) [inline src/docs/release-notes/R22/R22.mdx:19]: I assume you have a specific reason for not listing "Design Pattern Examples" under "Documentation Site"... I don't think people would know what this section is referring to from just the title.

**Review by huayunh — COMMENTED** [review]
Everything else looks good

File: src/docs/release-notes/R22/R22.mdx (hunk 2 above)
Comment (jeffvg) [reply src/docs/release-notes/R22/R22.mdx:19]: Yeah, we did have it under the documentation section and we didn't want readers to assume that these were just documentation on doc'it. We can put any label on it. What do you suggest

Comment (huayunh) [reply src/docs/release-notes/R22/R22.mdx:19]: interactive design pattern examples (they are called "interactive examples" on docit), design pattern code examples, etc

**Review by huayunh — APPROVED** [review]

**Pull request merged**